Documentation/cmdref/index_cilium.rst

```diff
@@ -0,0 +1,17 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+cilium Command Reference
```

Comment:
IMO it would help if we would name this ~ `cilium (CLI)` (including file (`index_cilium_cli.rst`) - even though the actual binary is named `cilium`

Reply:
Updated 👍